### PR TITLE
Additional event logging for add/delete/update informer events

### PR DESCRIPTION
**Pull request by abhat**

### CODE REVIEW

File: pkg/openshift-sdn/informers.go [hunk 1]
```diff
@@ -121,3 +125,31 @@ func applyClientConnectionOverrides(kubeConfig *rest.Config) {
...
+type eventHandlingLogger struct{}
+
+func describe(obj interface{}) string {
+	if svc, ok := obj.(*v1.Service); ok {
+		if svc.Spec.Type == v1.ServiceTypeExternalName {
+			return fmt.Sprintf("externalname service %s/%s", svc.Namespace, svc.Name)
+		} else if svc.Spec.ClusterIP == v1.ClusterIPNone {
+			return fmt.Sprintf("headless service %s/%s", svc.Namespace, svc.Name)
+		} else {
+			return fmt.Sprintf("service %s/%s", svc.Namespace, svc.Name)
```
Comment (alexanderConstantinescu) [inline pkg/openshift-sdn/informers.go]: What do you think about logging the entire object? We might realize later on that a couple of attributes are not enough to debug the issue completely?

Comment (abhat) [reply pkg/openshift-sdn/informers.go]: Sure, that makes sense. let me update the PR.

Comment (squeed) [reply pkg/openshift-sdn/informers.go]: yeah, agreed. I'd serialize the service object to json and log it.

Comment (abhat) [reply pkg/openshift-sdn/informers.go]: does it need serialization, I am logging it as `%+v`?

**Pull request merged**